primap2/_data_format.py

```diff
@@ -251,7 +261,7 @@ def ensure_entity_and_units_valid(key: Hashable, da: xr.DataArray):
                 f"compatible with an emission rate."
             )
     except pint.UndefinedUnitError:
-        if entity not in ("population",) and "(" not in entity:
+        if entity not in ("population",) and "(" not in key:
```

Comment:
Is this a workaround because population is not available in scmdata-units yet?

Reply:
That's a philosphical question. According to metrologists, "population" does not have a unit, so it is fine to be unit-less here. If scmdata-units should gain a unit suitable for the entity "population", we would of course use it.

Reply:
It would make sense to have the million etc prefixes though, I think.
In primap 1 units is "pers"